coordinator/src/node/resize.rs

```diff
@@ -340,6 +340,12 @@ fn compute_average_execution_price(
 ) -> Decimal {
     let starting_average_execution_price = decimal_from_f32(starting_average_execution_price);
 
+    if starting_direction != trade_direction {
```

Comment:
If he is selling more than he had before the execution price should update, i.e. if he had 20 contracts long and sold 30 short he is now 10 negative at the new price.

Reply:
https://github.com/get10101/10101/pull/1628#issue-2006431156

Reply:
> If he is selling more than he had before the execution price should update, i.e. if he had 20 contracts long and sold 30 short he is now 10 negative at the new price.

How is that average_execution_price calculated? In this context we only have on order matched by the bid / ask price, but we would actually need two orders don't we? one for the ask and one for the bid (and vice versa)